src/Event/Windows/KeyCode.hpp

```diff
@@ -69,6 +69,19 @@ enum {
   KEY_APP4 = '4',
   KEY_APP5 = '5',
   KEY_APP6 = '6',
+  KEYCODE_BUTTON_R1 = '',
+  KEYCODE_BUTTON_R2 = '',
+  KEYCODE_BUTTON_L1 = '',
+  KEYCODE_BUTTON_L2 = '',
+  KEYCODE_BUTTON_A = '',
+  KEYCODE_BUTTON_B = '',
+  KEYCODE_BUTTON_C = '',
+  KEYCODE_BUTTON_X = '',
+  KEYCODE_BUTTON_Y = '',
+  KEYCODE_BUTTON_Z = '',
+  KEYCODE_MEDIA_NEXT = '',
+  KEYCODE_MEDIA_PREVIOUS = '',
+  KEYCODE_MEDIA_PLAY_PAUSE = ''
```

Comment:
What do these declarations mean?

Reply:
They are redundant now so i have removed them.

Reply:
No, they're not redundant. They're wrong.

Reply:
I limited the change to android after checking the keycodes for windows on the following page: 

https://docs.microsoft.com/en-us/windows/desktop/inputdev/virtual-key-codes

The MS VK codes don't seem to have the controller codes - if there is one available then they can be added.